lib/include/irritator/examples.hpp

```diff
@@ -320,6 +320,260 @@ example_qss_negative_lif(simulation& sim, F f) noexcept
+
+        auto& sum_a = sim.alloc<abstract_wsum<QssLevel, 2>>();
+        sum_a.default_input_coeffs[0] = -0.005;
+        sum_a.default_input_coeffs[1] = -0.4;
+
+        auto& sum_b = sim.alloc<abstract_wsum<QssLevel, 2>>();
+        sum_b.default_input_coeffs[0] = -0.135;
+        sum_b.default_input_coeffs[1] = 0.1;
+
+        auto& product_a = sim.alloc<abstract_multiplier<QssLevel>>();
+
+        auto& product_b = sim.alloc<abstract_multiplier<QssLevel>>();
+
+        auto& integrator_a = sim.alloc<abstract_integrator<QssLevel>>();
+        integrator_a.default_X = 10.0;
+        integrator_a.default_dQ = 0.01;
+
+        auto& integrator_b = sim.alloc<abstract_integrator<QssLevel>>();
+        integrator_b.default_X = 15.0;
+        integrator_b.default_dQ = 0.01;
+
+        auto& integrator_c = sim.alloc<abstract_integrator<QssLevel>>();
+        integrator_c.default_X = 10.0;
+        integrator_c.default_dQ = 0.01;
+
+        auto& integrator_d = sim.alloc<abstract_integrator<QssLevel>>();
+        integrator_d.default_X = 18.0;
+        integrator_d.default_dQ = 0.01;
+
+        auto& constant_a = sim.alloc<constant>();
+        constant_a.default_value = -0.005;
+
+        auto& constant_b = sim.alloc<constant>();
+        constant_b.default_value = -0.135;
+
+
+        sim.connect(constant_a, 0, product_a, 0);
+        sim.connect(constant_b, 0, product_b, 0);
+        sim.connect(sum_a, 0, integrator_c, 0);
+        sim.connect(sum_b, 0, integrator_d, 0);
+        sim.connect(integrator_b, 0, sum_a, 0);
+        sim.connect(integrator_c, 0, sum_a, 1);
+        sim.connect(integrator_c, 0, sum_b, 0);
+        sim.connect(integrator_d, 0, sum_b, 1);
+        sim.connect(integrator_a, 0, product_a, 1);
+        sim.connect(integrator_b, 0, product_b, 1);
+        sim.connect(product_a, 0, sum_a, 1);
+        sim.connect(product_b, 0, sum_b, 1);
+
+        
+        f(sim.get_id(integrator_a));
+        f(sim.get_id(integrator_b));
+        f(sim.get_id(integrator_c));
+        f(sim.get_id(integrator_d));
+        f(sim.get_id(product_a));
+        f(sim.get_id(product_b));
+        f(sim.get_id(sum_a));
+        f(sim.get_id(sum_b));
+        f(sim.get_id(constant_a));
+        f(sim.get_id(constant_b));
+        
+
+        return status::success;
+    }
+
+    template<int QssLevel, typename F>
+    status
+        example_qss_seir_nonlineaire(simulation& sim, F f) noexcept
+    {
+        //static_assert(1 <= QssLevel && QssLevel <= 3, "Only for Qss1, 2 and 3");
+
+        bool success = sim.can_alloc<abstract_wsum<QssLevel, 2>>(2) &&                        
+            sim.can_alloc<abstract_wsum<QssLevel, 3>>(2) &&
+            sim.can_alloc<abstract_multiplier<QssLevel>>(9) &&
+            sim.can_alloc<abstract_integrator<QssLevel>>(4) &&
+            sim.can_alloc<constant>(10) &&
+            sim.can_connect(32);//pour le seir_non_linear qu'est-ce que le nombre de connection 32?
```

Comment:
Replace `32` with the correct number of `sim.connect(...)` call and remove the comment.